### PR TITLE
Fixes Phoron Salts Again

### PR DESCRIPTION
**Pull request by BurgerLUA**
Because I'm an idiot I realised that phoron salt bombs are actually impossible because water turns to ice when frozen, and you need frozen water in order to make a working bomb.

I also fixed a few bugs regarding cryo/pyro mixes in reagents. It only took me 5 hours to figure out.

I also made Phoron Salts containers spawn with a stable temperature so admins don't blow up when they mess around with it.

I adjusted the explosion size of phoron salts so that absolute best case scenario, you can make a bomb that has the destructive radius of a screen.

### CODE REVIEW

**Comment by BurgerLUA** [comment]
!review

**Comment by BurgerLUA** [comment]
Don't merge this yet. I'm going to add a few other things as well.

**Comment by BurgerLUA** [comment]
@Alberyk I also snuck in a balance change just now so you might want to review it to see if you're comfortable. The multiplier is set to a specific number so that if you manage to get the best mixture possible (will probably honestly take a shitload of reagents and time to make) you can make a bomb that has the total destructive radius of the screen (8 radius).

File: code/modules/reagents/Chemistry-Recipes.dm [hunk 1]
```diff
@@ -102,7 +102,8 @@
 	for(var/reactant in required_reagents)
 		var/amt_used = required_reagents[reactant] * reaction_progress
 		var/datum/reagent/removing_reagent = holder.get_reagent(reactant)
-		total_thermal_energy += removing_reagent.get_thermal_energy() * (amt_used / removing_reagent.volume)
```
Comment (Sindorman) [inline code/modules/reagents/Chemistry-Recipes.dm]: How did this work before?

Comment (BurgerLUA) [reply code/modules/reagents/Chemistry-Recipes.dm]: It's the same except its just put into a var for easy debugging.

Though I'm not sure I understand the question in the first place.

Comment (Sindorman) [reply code/modules/reagents/Chemistry-Recipes.dm]: never mind, I saw it wrong

**Pull request merged**